backend/curriculum_tracking/management/commands/set_bulk_due_dates.py

```diff
@@ -0,0 +1,27 @@
+from django.core.management.base import BaseCommand
+from curriculum_tracking.models import AgileCard, Team
+from ..helpers import get_team_cards
+
+
+class Command(BaseCommand):
+    """
+    Generic command structure: python manage.py set_bulk_due_dates "team_name" "content_item_id" "flavour" "YYYY-MM-DD"
+    command example: python manage.py set_bulk_due_dates bomb_squad 2 Python 2021-02-18
+    """
```

Comment:
Why do we have both a an endpoint and a command? If we've got one, isn't the other redundant?

Reply:
The instructions mentioned something about a command and something about an API endpoint, so I created both.  I can remove the command if it is not needed?

Reply:
I think this can be removed.

It's important to read instructions critically. 

Initially this issue was designed to allow a new person on the team to learn some useful skills while adding value. Managment scripts are easy to make as compared to api endpoints. But if we can access functionality via the api we dont need the managment script, it wont get used